ui/app/templates/partials/tools/unwrap.hbs

```diff
@@ -8,19 +8,40 @@
 
 {{#if unwrap_data}}
   <div class="box is-sideless is-fullwidth is-marginless">
-    <div class="field">
-      <label class="is-label">
-        Unwrapped data
-      </label>
-      <div class="control">
-        {{json-editor
-          value=(stringify unwrap_data)
-          options=(hash
-            readOnly=true
-          )
-        }}
+    <nav class="tabs">
+      <ul>
+        <li class="{{if (eq unwrapActiveTab "data") "is-active"}}">
+          <a href="#" {{action "updateUnwrapTab" "data"}} class="link link-plain has-text-weight-semibold {{if (eq unwrapActiveTab "data") "is-active"}}">Data</a>
+        </li>
+        <li class="{{if (eq unwrapActiveTab "details") "is-active"}}">
+          <a href="#" {{action "updateUnwrapTab" "details"}} class="link link-plain has-text-weight-semibold {{if (eq unwrapActiveTab "details") "is-active"}}">Wrap Details</a>
```

Comment:
You can use the `mut` helper here if don't want to add the action to the component file. Something like `{{action (mut unwrapActiveTab) "details"}}` - here's the docs for it https://www.emberjs.com/api/ember/3.5/classes/Ember.Templates.helpers/methods/mut?anchor=mut and we use it quite a bit if you search around the code base.